mappings/net/minecraft/entity/EntityLiving.mapping

```diff
@@ -8,9 +8,13 @@ CLASS sp net/minecraft/entity/EntityLiving
 	FIELD au LIVING_FLAGS Lkj;
 	FIELD aw preferredHand Lrb;
 	FIELD ay stuckArrowTimer I
+	FIELD az hurtTime I
```

Comment:
I got this name from the EntityLiving serialize/deserialize tags. I will remove the mapping since I don't understand it.